src/components/HomepageFeatures/index.js

```diff
@@ -13,9 +13,11 @@ function Feature({ Svg, to, title, description }) {
         </div>
       </Link>
       <div className="text--center padding-horiz--md">
-        <h3 className="py-5">
-          <strong>{title}</strong>
-        </h3>
+        <Link to={to}>
```

Comment:
We could do the whole div as well.

Reply:
Can't run it locally, but applied that change 👌